apps/backend/prompts/github/QA_REVIEW_SYSTEM_PROMPT.md

```
@@ -0,0 +1,192 @@
+## Questions for the Team
+1. [Question that needs human input]
+2. ...
+```
+
+---
+
+## ATTACH LOGS BELOW
+
+Paste the PR review debug logs here for analysis:
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)